examples/pytorch/question-answering/utils_qa.py

```diff
@@ -38,7 +38,7 @@ def postprocess_qa_predictions(
     null_score_diff_threshold: float = 0.0,
     output_dir: Optional[str] = None,
     prefix: Optional[str] = None,
-    is_world_process_zero: bool = True,
+    log_level: Optional[str] = "passive",
```

Comment:
this should be `int` and not `passive` default. Let's set the default to `logging.WARNING`

"passive" is a sort of leave defaults in place